nixos/modules/services/web-apps/mattermost.nix

```diff
@@ -7,7 +7,12 @@ let
   cfg = config.services.mattermost;
 
   defaultConfig = builtins.fromJSON (builtins.replaceStrings [ "\\u0026" ] [ "&" ]
-    (readFile "${pkgs.mattermost}/config/config.json")
+    # Use `unsafeDiscardStringContext` to support evaluation with Nix >= 2.6 where the
+    # string passed to `fromJSON` is not allowed to have a context.
+    # This is safe because `config.json` does not reference any store paths.
+    (builtins.unsafeDiscardStringContext
```

Comment:
Is this builtin available on Nix 2.3+ (asking specially because NixOS 21.11 still defaults to 2.3)?

Reply:
Yes:
```
nix shell nixpkgs/38346f64616c3176f73ad0f20e51557ec0f3d75d#nix_2_3 -c nix eval '(builtins ? unsafeDiscardStringContext)' #=> true
```